ros:
  node_name: "grasp_generator"  # Name of the ROS node

descriptor:
  model: "gpt-4o"
  prompt: "Describe the object in the image with emphasis on its shape and clearly defined features. Be concise and do not exceed 100 words."

generator:
  model: "dall-e-3"
  quality: "hd" # Only available for DALL-E 3 (set it otherwise to null)
  size: "1024x1024" # Output image size (width x height), possible values: "1024x1024", "1024x1792", "1792x1024"
  style: "natural"  # "vivid" or "natural" (only available for DALL-E 3)
  background_prompt: "The background is a simple uniform wall."

debug:
  out_dir: "/Handover_Pipeline/outputs/"
  log_description_prompt: True  # log prompt used for object description
  log_generation_prompt: True  # log prompt used for image generation